NEWS.rst

```diff
@@ -27,7 +27,8 @@ New module Bio.PDB.SASA implements the Shrake-Rupley algorithm to calculate
 atomic solvent accessible areas without third-party tools.
 
 Bio.UniProt.GOA now parses Gene Product Information (GPI) files version 1.2,
-files can be found on the EBI ftp site ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/
+files can be downloaded from the EBI ftp site:
+ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/
```

Comment:
If you wanted two lines, how about:

```
Bio.UniProt.GOA now parses Gene Product Information (GPI) version 1.2, files
can be found on the EBI ftp site ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/
```

Reply:
@peterjc just a rewording of the documentation? Sure.

Reply:
@peterjc Just a rewording of the documentation? Sure.

Reply:
I meant could you look at the entire pull request, adding v1.2 GPI parsing.

Reply:
Looks good to me. Can approve it now. If you want, I can test it, but won't be before the weekend.

Reply:
Testing would be great, even if a week away. Thanks!

Reply:
@peterjc , thank you for having a look at this, I'm having an issue with the pre-commit hooks, the 2 lines you suggested fails doc8, line too long, my git pre-commit is checking flake8 and black but doc8, that is why the initial commit failed as well, I'll try rolling it back.

@idoerg , thank you.

Cheers,

Reply:
@peterjc , git reset didn't work :( had to do another commit, if you could squash them it would look better, thank you.

Reply:
I must have miscounted, I thought that version would fit fine. Sorry.

Anyway, there's no problem doing a git merge here - everything is authored by you.